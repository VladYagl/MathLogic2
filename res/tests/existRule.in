Psi(x)->Phi, Betta, Alpha|-?xPsi(x)->Phi
Psi(x)->Phi
?xPsi(x)->Phi
