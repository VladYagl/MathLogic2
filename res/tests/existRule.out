(Psi(x))->(Phi),Betta|-(Alpha)->((?x(Psi(x)))->(Phi))
(Psi(x))->(Phi)
((Psi(x))->(Phi))->((Alpha)->((Psi(x))->(Phi)))
(Alpha)->((Psi(x))->(Phi))
(Alpha)->((Alpha)->(Alpha))
((Alpha)->((Alpha)->(Alpha)))->((Psi(x))->((Alpha)->((Alpha)->(Alpha))))
(Psi(x))->((Alpha)->((Alpha)->(Alpha)))
(Alpha)->(((Alpha)->(Alpha))->(Alpha))
((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Psi(x))->((Alpha)->(((Alpha)->(Alpha))->(Alpha))))
(Psi(x))->((Alpha)->(((Alpha)->(Alpha))->(Alpha)))
((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))
(((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))))->((Psi(x))->(((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))))
(Psi(x))->(((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))))
((Psi(x))->((Alpha)->((Alpha)->(Alpha))))->(((Psi(x))->(((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))))->((Psi(x))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))))
((Psi(x))->(((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))))->((Psi(x))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))))
(Psi(x))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))
((Psi(x))->((Alpha)->(((Alpha)->(Alpha))->(Alpha))))->(((Psi(x))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))))->((Psi(x))->((Alpha)->(Alpha))))
((Psi(x))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))))->((Psi(x))->((Alpha)->(Alpha)))
(Psi(x))->((Alpha)->(Alpha))
(Psi(x))->((Psi(x))->(Psi(x)))
(Psi(x))->(((Psi(x))->(Psi(x)))->(Psi(x)))
((Psi(x))->((Psi(x))->(Psi(x))))->(((Psi(x))->(((Psi(x))->(Psi(x)))->(Psi(x))))->((Psi(x))->(Psi(x))))
((Psi(x))->(((Psi(x))->(Psi(x)))->(Psi(x))))->((Psi(x))->(Psi(x)))
(Psi(x))->(Psi(x))
(Psi(x))->((Alpha)->(Psi(x)))
((Psi(x))->((Alpha)->(Psi(x))))->((Psi(x))->((Psi(x))->((Alpha)->(Psi(x)))))
(Psi(x))->((Psi(x))->((Alpha)->(Psi(x))))
((Psi(x))->(Psi(x)))->(((Psi(x))->((Psi(x))->((Alpha)->(Psi(x)))))->((Psi(x))->((Alpha)->(Psi(x)))))
((Psi(x))->((Psi(x))->((Alpha)->(Psi(x)))))->((Psi(x))->((Alpha)->(Psi(x))))
(Psi(x))->((Alpha)->(Psi(x)))
(Alpha)->((Psi(x))->(Phi))
((Alpha)->((Psi(x))->(Phi)))->((Psi(x))->((Alpha)->((Psi(x))->(Phi))))
(Psi(x))->((Alpha)->((Psi(x))->(Phi)))
((Alpha)->((Psi(x))->(Phi)))->((Alpha)->((Alpha)->((Psi(x))->(Phi))))
(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->((Alpha)->((Psi(x))->(Phi)))))->((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->((Alpha)->((Psi(x))->(Phi))))))
(Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->((Alpha)->((Psi(x))->(Phi)))))
((Psi(x))->((Alpha)->((Psi(x))->(Phi))))->(((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->((Alpha)->((Psi(x))->(Phi))))))->((Psi(x))->((Alpha)->((Alpha)->((Psi(x))->(Phi))))))
((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->((Alpha)->((Psi(x))->(Phi))))))->((Psi(x))->((Alpha)->((Alpha)->((Psi(x))->(Phi)))))
(Psi(x))->((Alpha)->((Alpha)->((Psi(x))->(Phi))))
((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi))))
(((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi)))))->((Psi(x))->(((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi))))))
(Psi(x))->(((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi)))))
((Psi(x))->((Alpha)->(Alpha)))->(((Psi(x))->(((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi))))))->((Psi(x))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi))))))
((Psi(x))->(((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi))))))->((Psi(x))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi)))))
(Psi(x))->(((Alpha)->((Alpha)->((Psi(x))->(Phi))))->((Alpha)->((Psi(x))->(Phi))))
(Alpha)->((Psi(x))->(Phi))
((Alpha)->((Psi(x))->(Phi)))->((Psi(x))->((Alpha)->((Psi(x))->(Phi))))
(Psi(x))->((Alpha)->((Psi(x))->(Phi)))
((Alpha)->(Psi(x)))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi)))
(((Alpha)->(Psi(x)))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi))))->((Psi(x))->(((Alpha)->(Psi(x)))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi)))))
(Psi(x))->(((Alpha)->(Psi(x)))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi))))
((Psi(x))->((Alpha)->(Psi(x))))->(((Psi(x))->(((Alpha)->(Psi(x)))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi)))))->((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi)))))
((Psi(x))->(((Alpha)->(Psi(x)))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi)))))->((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi))))
(Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi)))
((Psi(x))->((Alpha)->((Psi(x))->(Phi))))->(((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi))))->((Psi(x))->((Alpha)->(Phi))))
((Psi(x))->(((Alpha)->((Psi(x))->(Phi)))->((Alpha)->(Phi))))->((Psi(x))->((Alpha)->(Phi)))
(Psi(x))->((Alpha)->(Phi))
(?x(Psi(x)))->((Alpha)->(Phi))
(Alpha)->((Alpha)->(Alpha))
(Alpha)->(((Alpha)->(Alpha))->(Alpha))
((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))
((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))
(Alpha)->(Alpha)
(Alpha)->((?x(Psi(x)))->(Alpha))
((Alpha)->((?x(Psi(x)))->(Alpha)))->((Alpha)->((Alpha)->((?x(Psi(x)))->(Alpha))))
(Alpha)->((Alpha)->((?x(Psi(x)))->(Alpha)))
((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((?x(Psi(x)))->(Alpha))))->((Alpha)->((?x(Psi(x)))->(Alpha))))
((Alpha)->((Alpha)->((?x(Psi(x)))->(Alpha))))->((Alpha)->((?x(Psi(x)))->(Alpha)))
(Alpha)->((?x(Psi(x)))->(Alpha))
(?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x))))
((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->((Alpha)->((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x))))))
(Alpha)->((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))
(?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x))))
((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((Alpha)->((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x))))))
(Alpha)->((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))
((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x)))))
(((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x))))))->((Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x)))))))
(Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x))))))
((Alpha)->((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x))))))->(((Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x)))))))->((Alpha)->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x)))))))
((Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->(?x(Psi(x)))))->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x)))))))->((Alpha)->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x))))))
(Alpha)->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x)))))
((Alpha)->((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x))))))->(((Alpha)->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x))))))->((Alpha)->((?x(Psi(x)))->(?x(Psi(x))))))
((Alpha)->(((?x(Psi(x)))->(((?x(Psi(x)))->(?x(Psi(x))))->(?x(Psi(x)))))->((?x(Psi(x)))->(?x(Psi(x))))))->((Alpha)->((?x(Psi(x)))->(?x(Psi(x)))))
(Alpha)->((?x(Psi(x)))->(?x(Psi(x))))
(?x(Psi(x)))->((Alpha)->(Phi))
((?x(Psi(x)))->((Alpha)->(Phi)))->((Alpha)->((?x(Psi(x)))->((Alpha)->(Phi))))
(Alpha)->((?x(Psi(x)))->((Alpha)->(Phi)))
((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))
(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi)))))->((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))))
(Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi)))))
((Alpha)->((?x(Psi(x)))->((Alpha)->(Phi))))->(((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))))->((Alpha)->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))))
((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))))->((Alpha)->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi)))))
(Alpha)->((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))
((?x(Psi(x)))->(?x(Psi(x))))->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi))))
(((?x(Psi(x)))->(?x(Psi(x))))->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi)))))->((Alpha)->(((?x(Psi(x)))->(?x(Psi(x))))->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi))))))
(Alpha)->(((?x(Psi(x)))->(?x(Psi(x))))->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi)))))
((Alpha)->((?x(Psi(x)))->(?x(Psi(x)))))->(((Alpha)->(((?x(Psi(x)))->(?x(Psi(x))))->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi))))))->((Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi))))))
((Alpha)->(((?x(Psi(x)))->(?x(Psi(x))))->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi))))))->((Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi)))))
(Alpha)->(((?x(Psi(x)))->((?x(Psi(x)))->((Alpha)->(Phi))))->((?x(Psi(x)))->((Alpha)->(Phi))))
(?x(Psi(x)))->((Alpha)->(Phi))
((?x(Psi(x)))->((Alpha)->(Phi)))->((Alpha)->((?x(Psi(x)))->((Alpha)->(Phi))))
(Alpha)->((?x(Psi(x)))->((Alpha)->(Phi)))
((?x(Psi(x)))->(Alpha))->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi)))
(((?x(Psi(x)))->(Alpha))->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi))))->((Alpha)->(((?x(Psi(x)))->(Alpha))->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi)))))
(Alpha)->(((?x(Psi(x)))->(Alpha))->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi))))
((Alpha)->((?x(Psi(x)))->(Alpha)))->(((Alpha)->(((?x(Psi(x)))->(Alpha))->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi)))))->((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi)))))
((Alpha)->(((?x(Psi(x)))->(Alpha))->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi)))))->((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi))))
(Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi)))
((Alpha)->((?x(Psi(x)))->((Alpha)->(Phi))))->(((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi))))->((Alpha)->((?x(Psi(x)))->(Phi))))
((Alpha)->(((?x(Psi(x)))->((Alpha)->(Phi)))->((?x(Psi(x)))->(Phi))))->((Alpha)->((?x(Psi(x)))->(Phi)))
(Alpha)->((?x(Psi(x)))->(Phi))
